### PR TITLE
Replaced the calls to io.join_paths with the standard os.path.join

### PR DESCRIPTION
**Pull request by koushikjoshi**

### CODE REVIEW

**Comment by smathermather** [comment]
Aha, for https://github.com/OpenDroneMap/ODM/issues/1177. Nice!

File: run.py [hunk 1]
```diff
@@ -32,15 +32,15 @@
 
         # Don't leak token
         if k == 'sm_cluster' and args_dict[k] is not None:
-            log.ODM_INFO('%s: True' % k)
+            log.ODM_INFO('%s: True' % k
```
Comment (pierotofy) [inline run.py]: I think this parenthesis was removed by mistake?

**Comment by pierotofy** [comment]
Thanks @koushikjoshi ! See my comment for a small change needed to merge this.

File: run.py (hunk 1 above)
Comment (koushikjoshi) [reply run.py]: @pierotofy yes you are right. I'm sorry, this was my first open-source contribution. Do you want me to add the paranthesis?

**Comment by pierotofy** [comment]
Looks great! Thanks @koushikjoshi :clap:

**Pull request merged**